Source/Core/Core/Lua/LuaFunctions/LuaBitFunctions.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <string>
+#include "src/lua.hpp"
+#include "src/lua.h"
+#include "src/luaconf.h"
+#include "src/lapi.h"
+#include "../LuaHelperClasses/LuaColonCheck.h"
+
+namespace Lua
+{
+namespace LuaBit
+{
+
+void InitLuaBitFunctions(lua_State* luaState, const std::string& luaApiVersion);
```

Comment:
Our convention is to use `snake_case` for variable names rather than `camelCase`, even when the utilized libraries use otherwise. This seems to have also trickled over into the UI-related files too.